File: school/algo/algo_date4/ex5-1.c
#include<stdio.h>

int main(){
	int birthmonth;
	printf("あなたの生まれた月を入力してください。:");
	scanf("%d",&birthmonth);

	printf("\n%d月は、",birthmonth);
	switch((birthmonth%12)/3){
		//四季に変換する。春は3-5月、夏は6-8月、秋は9-11月、冬は12-2月。
		case 0:
			printf("冬です。\n");
			//break;
		case 1:
			printf("春です。\n");
			//break;
		case 2:
			printf("夏です。\n");
			//break;
		case 3:
			printf("秋です。\n");
			//break;
		default:
			printf("理解できません。\n");
	}
	return(0);
}

File: school/com02/ex5-1.py
birthmonth = int(input("あなたの生まれた月を入力してください："))
print(str(birthmonth)+"月は、",end="")

#四季に変換して判定する。春は3-5月、夏は6-8月、秋は9-11月、冬は12-2月
if int((birthmonth%12)/3) == 0:
    print("冬です。")
elif int((birthmonth%12)/3) == 1:
    print("春です。")
elif int((birthmonth%12)/3) == 2:
    print("夏です。")
elif int((birthmonth%12)/3) == 3:
    print("秋です。")
else:
    print("理解できません。")